test/gen-cert

```diff
@@ -63,7 +63,12 @@ cat >> req.conf <<eof
 [ v3_req ]
 basicConstraints = CA:FALSE
 keyUsage = nonRepudiation, digitalSignature, keyEncipherment, keyCertSign
-subjectAltName = DNS.1:${DOMAIN}
+#subjectAltName = DNS.1:${DOMAIN}
```

Comment:
Unfortunately, I encountered the following issue with ssl cert with "*openstack.example.com" although it works well for other services on controller:
```
# swift list
hostname 'swift.openstack.example.com' doesn't match '*openstack.example.com'
```

So I have to revert this part to use my previous approach with 2 lines for Subject Alternative Names in the ssl cert generated by `gen-cert` script:
```
subjectAltName = @alt_names

[alt_names]
DNS.1 = ${DOMAIN}
DNS.2 = swift.${DOMAIN}
```